crates/smartmodule-development-kit/src/generate.rs

```diff
@@ -166,6 +176,10 @@ pub struct GenerateOpt {
         "SmCrateSourceGit", "SmCrateSourceCratesIo", "SmCrateSourcePath"],)]
     develop: bool,
 
+    /// Set the remote URL for the hub
+    #[clap(long, hide_short_help = true)]
+    hub_remote: Option<String>,
```

Comment:
remove this.  use env var

Reply:
It is hidden from users, and only used in testing. An env var didn't behave well in the CI testing

Reply:
ok. fair enough